src/Tec/Settings.php

```diff
@@ -204,49 +198,76 @@ public function delete_option( $key = '' ) {
 		return Tribe__Settings_Manager::set_options( $options );
 	}
 
-	/**
-	 * Here is an example of removing settings from Events > Settings > General tab > "Map Settings" section
-	 * that are specific to Google Maps.
-	 *
-	 * TODO: Remove this method and the corresponding hook in `__construct()` if you don't want to remove any settings.
-	 */
-	public function remove_settings() {
-		// Remove "Enable Google Maps" checkbox
-		$this->settings_helper->remove_field( 'embedGoogleMaps', 'general' );
-
-		// Remove "Map view search distance limit" (default of 25)
-		$this->settings_helper->remove_field( 'geoloc_default_geofence', 'general' );
-
-		// Remove "Google Maps default zoom level" (0-21, default of 10)
-		$this->settings_helper->remove_field( 'embedGoogleMapsZoom', 'general' );
-	}
-
 	/**
 	 * Adds a new section of fields to Events > Settings > General tab, appearing after the "Map Settings" section
 	 * and before the "Miscellaneous Settings" section.
-	 *
-	 * TODO: Move the setting to where you want and update this docblock. If you like it here, just delete this TODO.
 	 */
 	public function add_settings() {
+		$ticket_fieldsets = $this->get_ticket_fieldsets();
+
 		$fields = [
-			// TODO: Settings heading start. Remove this element if not needed. Also remove the corresponding `get_example_intro_text()` method below.
-			'Example'   => [
+			'default-fieldset-heading' => [
 				'type' => 'html',
-				'html' => $this->get_example_intro_text(),
+				'html' => $this->get_defaiult_fieldset_intro_text(),
 			],
-			// TODO: Settings heading end.
-			'a_setting' => [ // TODO: Change setting.
-				'type'            => 'text',
-				'label'           => esc_html__( 'Example setting', '__TRIBE_DOMAIN__' ),
-				'tooltip'         => sprintf( esc_html__( 'Example setting description. Enter your custom URL, including "http://" or "https://", for example %s.', '__TRIBE_DOMAIN__' ), '<code>https://demo.theeventscalendar.com/</code>' ),
-				'validation_type' => 'html',
+			'rsvp_default_fieldset'    => [
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'RSVP', 'tec-labs-default-ticket-fieldset' ),
+				'tooltip'         => sprintf( esc_html_x( 'The Ticket Fieldset to be added when an RSVP is created.', 'Setting description', 'tec-labs-default-ticket-fieldset' ) ),
+				'validation_type' => 'options',
+				'options'         => $ticket_fieldsets,
 			],
 		];
 
+		if ( class_exists( 'WooCommerce' ) ) {
+			$fields['wooticket_default_fieldset'] = [
+				'type'            => 'dropdown',
+				'label'           => esc_html__( 'WooCommerce ticket', 'tec-labs-default-ticket-fieldset' ),
+				'tooltip'         => sprintf(
+										// Translators: %s Name of the eCommerce platform.
+										esc_html_x(
+											'The Ticket Fieldset to be added when a ticket with %s is created.',
+											'Setting description',
+											'tec-labs-default-ticket-fieldset'
+										),
+										'WooCommerce'
+				),
```

Comment:
Do we need to use `sprintf` here? We're using `WooCommerce` as a string and not a variable.

Reply:
Nevermind, I just saw the EDD stuff after this.